cnxarchive/utils/ident_hash.py

```diff
@@ -29,6 +29,8 @@ class IdentHashError(Exception):
 
 class IdentHashSyntaxError(IdentHashError):
     """Raised when the ident-hash syntax is incorrect."""
+    def __init__(self, ident_hash):
+        self.ident_hash = ident_hash
```

Comment:
The ``ident_hash`` needs added to ``self.args``, otherwise It won't end up in the traceback. I know this because I've also made this change in an uncommited branch. :stuck_out_tongue: (see also, [the BaseException interface](https://docs.python.org/3.6/library/exceptions.html#BaseException))